src/os/vxworks/CMakeLists.txt

```diff
@@ -45,6 +45,19 @@ else ()
     )
 endif ()
 
+if (CMAKE_SYSTEM_VERSION VERSION_GREATER_EQUAL 7.0)
+    list(APPEND VXWORKS_IMPL_SRCLIST
+        ../portable/os-impl-posix-dirs.c
+    )
+    include_directories(${CMAKE_CURRENT_SOURCE_DIR}/inc/vxworks7)
+    add_definitions(-DOSAL_VXWORKS7)
+else ()
+    list(APPEND VXWORKS_BASE_SRCLIST
+        src/os-impl-dirs.c
+    )
+    include_directories(${CMAKE_CURRENT_SOURCE_DIR}/inc/vxworks6)
+    add_definitions(-DOSAL_VXWORKS6)
+endif ()
```

Comment:
For consistency I recommend appending to the `VXWORKS_IMPL_SRCLIST` not `VXWORKS_BASE_SRCLIST`

Also those "include_directories" don't seem to exist, and the OSAL_VXWORKS6/OSAL_VXWORKS7 define shouldn't be needed (see my other comment).

Reply:
Yeah that was a mistake. I will change it.

Reply:
the include_directories were leftover from the first attempt. I have removed them.